docs/lesson-plans/week-1.md

```diff
@@ -102,42 +95,541 @@ every week may not end up helping you as much as it would for our project, and y
+
+#### Backend
+
+pom.xml is a file with project build instructions, dependencies, and other project information.
+
+api/ is a directory handling routing information and data transfer objects (DTOs).
+
+persist/ is a directory for handling and saving data.
+
+service/ is the entry point for your project, so you'll start it up from there. It handles
+connecting everything together and basically 'running' your project.
+
+### What will each week look like?
+
+Every week, we'll have one of these meetings where we'll go over new concepts that
+you'll need to build your project. After the meetings, we may give out a description, starter
+code, and other resources for what we think you should work on during the upcoming week. 
+You're always free to work on other parts of your project or not do it at all if that's
+what you want to do. The following weekend before the next meeting, we'll send out
+what we're calling 'catch-up code', which is our implementation of the concepts and
+descriptions we told you to work on during the week. This catch-up code will show
+the current state of the project and the progression of our app over time. The
+reason we're doing this is because we understand that there are weeks that you may 
+be sick, drowning in work, just not in the mood to work on your project, or stuck on some 
+issues, but you'll probably want to stay caught up with the material. This is our way of 
+ensuring everyone has the opportunity to be on the same page.
+
+### Important Links
+
+Just in case you don't have the links or dont't know where they are, here are some 
+important course resources.
+
+>Note: there is also a dictionary of common terms we've compiled for you which should be
+>available in the Jumpstart Docs
+[C4C Jumpstart Docs](https://learn.c4cneu.com) (where this should be)
+
+[C4C Slack](https://c4cneu.slack.com/)
+
+[C4C GitHub Repository](https://github.com/Code-4-Community)
+
+[C4C Website](https://c4cneu.com)
+
+[C4C Dev Docs](https://docs.c4cneu.com)
+
+#### Making edits to the dictionary
+
+If there's a term you'd like to see added to the dictionary, updated, or formatted differently, you
+can click on the pencil icon in the upper right to make an edit (you can do this to the script pages
+too if there's a change you want there instead). You will probably need an account, 
+and then it should take you to an in-browser edit page. Once you're ready to make the change,
+you should be able to go to the bottom to leave a quick message detailing what you changed
+(you don't have to do the extended description). Then you'll be able to create a branch, and we'll
+review it to make sure it's a good edit, and let you merge it in. Then you'll have successfully 
+contributed to the C4C dictionary.
+
+## Project Setup
+
+Now we'll be setting up your project. If you want to see videos for setting up the 
+frontend and backend, see below.
+
+The overall setup process will just involve cloning the repos on your computer and 
```

Comment:
I think we should explain/define "cloning" and "repos" first, to someone unfamiliar with Git this will sound very jargony.

Reply:
Added an explanation.